### PR TITLE
Quick fixes

### PR DESCRIPTION
**Pull request by mgale456**
These are some quick fixes as a result of the great merge during the last meeting. Hopefully everyone can pull and save off a copy of the repo in a working condition before we break it again. 

AFAIK everything works, but I didn't do exhaustive testing.

@lpulley note that I reverted the django Dockerfile to using the Python-3 container instead of the smaller slim-buster version. I still like that idea, but it wasn't working for me right now.

### CODE REVIEW

File: django/Dockerfile [hunk 1]
```diff
@@ -1,21 +1,9 @@
...
-
-RUN mkdir -p /install
-COPY requirements.txt /
-RUN pip install --no-warn-script-location --prefix=/install -r /requirements.txt
-
-
-
-FROM python:3-slim-buster
-
-ENV PYTHONDONTWRITEBYTECODE 1
```
Comment (lpulley) [inline django/Dockerfile:11]: You should leave this `ENV` line in.

**Review by lpulley — CHANGES_REQUESTED** [review]

File: django/Dockerfile (hunk 1 above)
Comment (mgale456) [reply django/Dockerfile:11]: I added it back in. Also learned something :)

**Review by lpulley — APPROVED** [review]

File: django/Dockerfile (hunk 1 above)
Comment (lpulley) [reply django/Dockerfile:11]: It makes the image smaller. Now, using gunicorn & uvicorn, there will be multiple worker processes, and each will compile its own bytecode, which will be a slight performance hit on initial load, but I think it's fine? Worth testing image size vs. performance later.

**Review by MGhanayem63 — APPROVED** [review]
The server works fine.

**Pull request merged**